test/common/markdown/markdown.tests.ps1

```diff
@@ -0,0 +1,107 @@
+$moduleRootFilePath = Split-Path -Path $PSScriptRoot -Parent
+
+# Identify the repository root path of the resource module
+$repoRootPath = (Resolve-Path -LiteralPath (Join-path $moduleRootFilePath "../..")).ProviderPath
+Write-Verbose -message "RepoRoot: $repoRootPath" -Verbose
+Write-Verbose -message "PSScriptRoot: $PSScriptRoot" -Verbose
+$repoRootPathFound = $false
+
+Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
+    if (Get-Command -Name 'npm' -ErrorAction SilentlyContinue)
```

Comment:
The error should not lead to ignoring of the block so we need use "Skip".
It seems meaningless to perform this test on _both_ platforms but the number of tests on both platforms should be the same.
```powershell
BeforeAll {
    $skip = ! $isWindows
    if ( !$skip ) {
        if (Get-Command -Name 'npm' -ErrorAction SilentlyContinue) {
            $NpmInstalled = "Installed"
            ...
        } else {
            $NpmInstalled = "We're using this tool to delete the node_modules folder because it gets too long ..."
        }
    }
AfterAll {
    if (!$skip) {
        $null = Start-Process ...
    }
}
    It "Should not have errors in any markdown files" -Skip:$skip {
        $NpmInstalled | Should BeExactly "Installed"
        ...
    }
}